gateway/webhook/webhook.go

```diff
@@ -334,6 +335,13 @@ func (bt *batchWebhookTransformerT) batchTransformLoop() {
 				req.done <- transformerResponse{Err: response.GetStatus(response.InvalidJSON)}
 				continue
 			}
+			if len(body) > bt.webhook.config.maxReqSize.Load() {
```

Comment:
How costly is `Load` function? would it be okay to keep it outside the loop? If it's not costly, should be fine.

Reply:
It's just map lookup. https://github.com/rudderlabs/rudder-go-kit/blob/main/config/hotreloadable.go#L784-L790